autowarefoundation/autoware.universe
# feat(hazard_status_converter): add package

## Description

https://github.com/orgs/autowarefoundation/discussions/4176
This change removes `system_error_monitor` and `/hazard_status` is no longer output, so add a conversion node for compatibility.

Note: This node was first implemented in https://github.com/tier4/autoware.universe. Therefore, the publication year of the copyright is 2023.

## Tests performed

1. Run this command
    `ros2 launch diagnostic_graph_aggregator example.launch.xml`
2. Run this command in other terminal
    `ros2 launch hazard_status_converter hazard_status_converter.launch.xml`
3. Check if hazard status is output.
    `ros2 topic echo /hazard_status`

## Effects on system behavior

None

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
